src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,118 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import ClassNames from 'classnames';
```

Comment:
это функция, с маленькой буквы
```suggestion
import classNames from 'classnames';
```